### DISCUSSION TITLE
Annovar to maf

### DISCUSSION THREAD

**Discussion by shhabnsbwjd** [PoisonAlien/maftools#844] [Q&A]
Hello,

I am trying to import my variant file with all the required maf columns in R studio 'Chr Start End Ref Alt Gene.refGene GeneDetail.refGene ExonicFunc.refGene AAChange.refGene Tumor_Sample_Barcode Func.refGene' using this code; 
var.annovar2 <- system.file("extdata", "mutation.hg19_multianno.txt", package = "maftools") 

But imported file is empty

Kindly can someone help me please what am missing. I really need help

Thank you

**Comment by PoisonAlien** [comment]
Hi,

You dont have to use `system.file()`. It is only required to access the example file bundled with the maftools package.
You can directly use the `annovarToMaf()` to process your `mutation.hg19_multianno.txt` file.

```r
annovarToMaf(annovar = "mutation.hg19_multianno.txt")
```

**Reply by shhabnsbwjd** [reply]
It worked.... Thank you so much. And what if I want to use my vcf file??

**Reply by PoisonAlien** [reply]
VCF files are not supported for now. You will have to convert them into MAF. Checkout vcf2maf.

**Comment by shhabnsbwjd** [comment]
Thank you so much

On Tue, 5 Jul 2022 at 04:42, Anand Mayakonda ***@***.***>
wrote:

> Hi,
>
> You dont have to use system.file(). It is only required to access the
> example file bundled with the maftools package.
> You can directly use the annovarToMaf() to process your
> mutation.hg19_multianno.txt file.
>
> annovarToMaf(annovar = "mutation.hg19_multianno.txt")
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PoisonAlien/maftools/discussions/844#discussioncomment-3081577>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZ5F24F3YTRFVWFH64FNM63VSOVKHANCNFSM52TO4HBQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
*Maryam Shah*
PhD Student,
Peninsula Medical School,
Faculty of Health,
University of Plymouth, UK
